freestanding/dune

```diff
@@ -0,0 +1,16 @@
+(copy_files#
+ (alias install)
```

Comment:
while I know I've been using this as well, the `alias install` is wrong. what is actually missing are the correct dependencies in the rule below (`(deps Makefile YYY)` where `YYY` should be all C sources) -- unfortunately this means we need to name the C sources in both dune and Makefile (so adding a new C file is quite some bookkeeping).